config/routes.rb

```diff
@@ -1,22 +1,17 @@
 Rails.application.routes.draw do
+  get '/landing' => 'welcome#show', id: 'landing'
+
   post "oauth/callback" => "oauths#callback"
   get "oauth/callback" => "oauths#callback"
   get "sign_in/:provider" => "oauths#oauth", as: 'auth_at_provider'
   delete 'sign_out' => 'user_sessions#destroy'
 
-  get 'landing' => 'pages#show', id: 'landing'
-  get '/wikis' => 'wikis#index', as: 'wiki_index'
-  get '/:id' => 'wikis#show', as: 'wiki'
-  put '/:id' => 'wikis#update', as: 'edit_wiki'
-  post '/new' => 'wikis#new', as: 'new_wiki'
-  delete '/:id' => 'wikis#destroy', as: 'delete_wiki'
-  get '/:wiki_id/pages/:id' => 'pages#show', as: 'wiki_page'
-  put '/:wiki_id/pages/:id' => 'pages#update', as: 'edit_wiki_page'
-  post '/:wiki_id/pages/new' => 'pages#new', as: 'new_wiki_page'
-  delete '/:wiki_id/pages/:id' => 'pages#destroy', as: 'delete_wiki_page'
+  resources :users, only: [:show, :edit]
 
-  get '/' => 'users#show', as: 'user'
-  root 'pages#show', id: 'landing'
+  resources :wikis, only: [:create, :index]
+  resources :wikis, path: '', only: [:show, :update, :destroy] do
+    resources :pages, only: [:create, :show, :update, :destroy]
+  end
 
-  resources :user, only: [:edit]
+  root 'welcome#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.